### PR TITLE
user guides: mining: Rewrite mining guide

### PR DESCRIPTION
**Pull request by 00-matt**

### CODE REVIEW

File: _i18n/en/resources/user-guides/mine-to-pool.md [hunk 1]
```diff
@@ -1,107 +1,194 @@
...
-There are many pools to choose from, a list is available at
-[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
-more frequent payouts, but mining on a smaller pool helps to keep the network
-decentralized.
+Before starting, you already need to have a wallet configured and
+working. The pool needs to know your wallet address to be able to send
+payments there.
 
-## Selecting a CPU miner
+<!-- TODO: Link to guide. -->
```
Comment (00-matt) [inline _i18n/en/resources/user-guides/mine-to-pool.md:9]: Wasn't sure which guide to link to here, or how to do it. The [Accepting Monero](https://web.getmonero.org/get-started/accepting/) guide and [Getting started with the CLI wallet](https://web.getmonero.org/resources/user-guides/monero-wallet-cli.html) both seemed like good choices.

File: _i18n/ar.yml [hunk 2]
```diff
@@ -422,7 +422,7 @@ user-guides:
   purchasing-storing: شراء وتخزين مونيرو بأمان
   verify-allos: تحقق من ملفات التثبيت علي ويندوز ولينُكس (متقدم)
   verify-windows: تحقق من الملفات علي ويندوز (للمبتدئين)
-  mine-on-pool: كيفيه التعدين في حقل (تجمّع للتعدين) مع برنامج xmr-stak-cpu
+  mine-on-pool: How to mine on a pool
```
Comment (erciccione) [inline _i18n/ar.yml:425]: Please leave the string empty for all languages (except English)

File: _i18n/de.yml [hunk 3]
```diff
@@ -423,7 +423,7 @@ user-guides:
   purchasing-storing: Monero sicher kaufen und verwahren
   verify-allos: Dateien auf Linux, Mac bzw. Windows in der Befehlszeile verifizieren (fortgeschritten)
   verify-windows: Dateien unter Windows verifizieren (Anfänger)
-  mine-on-pool: Mining auf einem Pool mit xmr-stak-cpu
+  mine-on-pool: How to mine on a pool
```
Comment (erciccione) [inline _i18n/de.yml:426]: This can be `Mining auf einem Pool`

File: _i18n/en/resources/user-guides/mine-to-pool.md [hunk 4]
```diff
@@ -114,3 +201,21 @@ You need to edit your system's group policies to enable locking large pages. Her
...
+	sudo sysctl -w vm.nr_hugepages=1168
+	sudo ./xmrig
+
+You may have to increase 1168 depending on how many NUMA nodes your
+CPU(s) have.
+
+## Getting Help
+
+An active Monero mining community on Reddit is
+[/r/MoneroMining](https://www.reddit.com/r/MoneroMining/). You can
```
Comment (erciccione) [inline _i18n/en/resources/user-guides/mine-to-pool.md:219]: We don't link to /r/moneromining from getmonero. See: https://repo.getmonero.org/monero-project/monero-site/-/merge_requests/1195. Better point to /r/monerosupport or whatever.

File: _i18n/en.yml [hunk 5]
```diff
@@ -494,7 +494,7 @@ user-guides:
   purchasing-storing: Securely purchasing and storing Monero
   verify-allos: Verify binaries on Linux, Mac, or Windows command line (advanced)
   verify-windows: Verify binaries on Windows (beginner)
-  mine-on-pool: How to mine on a pool with xmr-stak-cpu
+  mine-on-pool: How to mine on a pool
```
Comment (erciccione) [inline _i18n/en.yml:497]:
```suggestion
  mine-on-pool: How to mine on a pool with XMRig
```
Since the guide is specific to xmrig, maybe would be better to specify it in the titlle.

File: _i18n/en/resources/user-guides/mine-to-pool.md (hunk 1 above)
Comment (erciccione) [reply _i18n/en/resources/user-guides/mine-to-pool.md:9]: I think linking to *Accepting Monero* would be fine. You can link to it using `{{ site.baseurl }}/get-started/accepting/`

File: _i18n/en/resources/user-guides/mine-to-pool.md [hunk 6]
```diff
@@ -1,107 +1,194 @@
...
+
+Find the line in config.json that says `algo: null` and change it to
+`algo: "rx/0"`. By default, XMRig expects the pool to tell it which
+hashing algorithm to use.
+
+### Huge Pages 0%
+
+#### Allowing large pages on Windows
+
+Taken from xmr-stak-cpu's documentation:
```
Comment (erciccione) [inline _i18n/en/resources/user-guides/mine-to-pool.md:191]: Why was this added? Is the text below extracted from xmr-stak-cpu's documentation? If that's the case i would just rewrite it. It's known that fireice is hostile to Monero, i don't want to give him an excuse to complain or any visibility.

Comment (00-matt) [reply _i18n/en/resources/user-guides/mine-to-pool.md:191]: It was in the original guide and I think I took it from xmr-stak-cpu's docs. Since it's just copied verbatim from [the MSDN](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15) I'll get rid of the xmr-stak-cpu mention.

File: _i18n/en/resources/user-guides/mine-to-pool.md [hunk 7]
```diff
@@ -1,111 +1,197 @@
...
-## Selecting a pool
+## Wallet
 
-There are many pools to choose from, a list is available at
-[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
-more frequent payouts, but mining on a smaller pool helps to keep the network
-decentralized.
+Before starting, you already need to have a wallet configured and
+working. The pool needs to know your wallet address to be able to send
+payments there. See the [Accepting Minero guide]({{ site.baseurl
```
Comment (erciccione) [inline _i18n/en/resources/user-guides/mine-to-pool.md:7]: `Minero`. Same for all languages

Comment (00-matt) [reply _i18n/en/resources/user-guides/mine-to-pool.md]: Fixed, also got rid of the TODO comment.

**Comment by 00-matt** [comment]
Sorry, somehow I totally forgot about what we talked about in the issue (about Tor). Just mentioned it with a link to the XMRig docs at the bottom of the guide.

**Comment by erciccione** [comment]
@00-matt this needs to be rebased

**Comment by 00-matt** [comment]
Rebased.
